gm4_washing_tanks/pack.mcmeta

```diff
@@ -0,0 +1,32 @@
+{
+    "pack": {
+        "pack_format": 5,
+        "description": "A Gamemode 4 Module"
+    },
+    "module_name": "Washing Tanks",
+    "module_id": "washing_tanks",
+    "creation_meta":"Created using the Gamemode 4 Module Template Generator (gm4.co/modules/generator)",
+    "site_description": "Dye! Die!",
```

Comment:
Should this be "Die! Dye!" instead? As in kill the dyes?

Reply:
yes